dagshub/data_engine/model/query_result.py

```diff
@@ -490,7 +491,9 @@ def predict_with_mlflow_model(
 
         Args:
             repo: repository to extract the model from
-            name: name of the model in the mlflow registry
+            name: name of the model in the mlflow registry. \
+                    Supported model flavors: `torch`, `tensorflow`, `pyfunc`, `scikit-learn`. \
+                    Any model that uses the `self.predict` API is also supported. \
```

Comment:
This shouldn't be under the name argument, and on the top level docs of the function instead.
Also sphinx's docs use double backticks for inline code instead of single backticks